.github/workflows/build.yaml

```diff
@@ -73,6 +75,7 @@ jobs:
           platforms: ${{ env.platforms }}
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+          annotations: ${{ steps.meta.outputs.labels }}
```

Comment:
@crazy-max Thanks for pointing it out! (It was fixed in a follow-up PR #652, but I still appreciate it very much.)

Reply:
@crazy-max BTW, in case you are digging into this, I partially reverted #652 in yet another PR #659 by adding annotations only within the index. I feel the support of annotations (if any) is poorly documented across registries---hopefully things will become better over the years.